modules/ossm-supported-configurations.adoc

```diff
@@ -18,4 +19,13 @@ OpenShift Online and OpenShift Dedicated are not supported for {ProductName} {Pr
 * This release of {ProductName} is only available on {product-title} x86_64.
 * {ProductName} is only suited for {product-title} Software Defined Networking (SDN) configured as a flat network with no external providers.
```

Comment:
I'm not sure what we officially support, but I believe we should be able to support all SDN modes: NetworkPolicy, Multitenant, and Subnet (flat network).

Reply:
@knrc Can you clarify this for GA?

Reply:
Given the short amount of time left, this can be addressed post-GA.